spec/integration/client/client_spec.rb

```diff
@@ -306,6 +306,7 @@ class ::Blah
 cookbook_path "#{path_to('cookbooks')}"
 EOM
       result = shell_out!("#{chef_client} -c \"#{path_to('config/client.rb')}\" -o 'x::default' -z --profile-ruby", :cwd => chef_dir)
+      result.error!
```

Comment:
My recollection is that `shell_out!` with the `!` should implicitly call `error!`